db_revisions/versions/31e39764e3c0_create_contact_info_table.py

```diff
@@ -32,7 +32,7 @@ def upgrade() -> None:
         sa.Column("hq_address_state", sa.String, nullable=False),
         sa.Column("hq_address_zip", sa.String, nullable=False),
         sa.Column("email", sa.String, nullable=False),
-        sa.Column("phone", sa.String, nullable=False),
+        sa.Column("phone_number", sa.String, nullable=False),
```

Comment:
I think a separate script to alter the column name.  This requires a full drop of the existing tables and scripts to pick up this change and don't think that is desireable.